internal/dinosaur/pkg/config/central_request.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+import (
+	"time"
+
+	"github.com/spf13/pflag"
+)
+
+// CentralRequestConfig holds all configuration for CentralRequests, e.g. expiration timeouts.
+type CentralRequestConfig struct {
+	CentralRequestExpirationTimeout  time.Duration `json:"central_request_expiration_timeout"`
+	CentralRequestInternalUserAgents []string      `json:"central_request_internal_user_agents"`
```

Comment:
```suggestion
	ExpirationTimeout  time.Duration `json:"central_request_expiration_timeout"`
	InternalUserAgents []string      `json:"central_request_internal_user_agents"`
```
The prefix is a bit redundant imo.